pom.xml

```diff
@@ -196,6 +196,11 @@
         <artifactId>flyway-core</artifactId>
         <version>${flyway.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.flywaydb</groupId>
+        <artifactId>flyway-database-postgresql</artifactId>
```

Comment:
Forstår jeg det rigtigt, som at vi nu skal depende på flyway-database-postgresql i stedet for flyway-core? I så fald skal vi have beskrevet det i CHANGELOG.md, for så er det vel en breaking change, som vil ramme mange Java projekter?

Reply:
PostgreSQLConfigurationExtension som bliver brugt i sample migreringen ligger nu i en postgres specifik artifact.

Hvis ikke man skal bruge den slags tror jeg muligvis godt man kan nøjes med flyway-core, men her skal flyway-database-postgresql på som compile scope.

Det skal jeg gerne teste med dataio, pt. er den version jeg har bygget baseret på den gamle super-pom, da den skulle kunne bygge på is'en.